ONSdigital/eq-questionnaire-runner
# Update mutually exclusive options to use multiple

### What is the context of this PR?
Updates Runner to use radio buttons when there is more than one mutually exclusive answer to a question.

Also updates some test text to make it make more sense

### How to review 
- Use the `test_mutually_exclusive_multiple` schema to test questions with multiple mutually exclusive options
- Use the `test_mutually_exclusive` schema to test questions with single mutually exclusive options still work as expected

### Checklist
